### PR TITLE
Implement inactivated account expiry

### PR DESCRIPTION
**Pull request by marto97**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -24,11 +24,12 @@
     ]
   },
   "include": [
-    "./src/**/*.ts"
+    "./src/**/*.ts",
+    "src/scs-jobs/delete_expired_users_job.ts"
```
Comment (Arun-KumarH) [inline tsconfig.json:28]: This is not needed since src recursively included

File: src/utils.ts [hunk 2]
```diff
@@ -266,3 +273,148 @@ export async function fetch(...args) {
...
+    picked.data = _filterJobData(picked.data, false, logger);
+    if (picked?.data?.payload?.value) {
+      picked.data.payload.value = Buffer.from(picked.data.payload.value);
+    }
+  }
+
+  return picked as any;
+}
+
+export async function runWorker(queue: string, concurrency: number, cfg: any, logger: Logger, events: Events, cb: Processor): Promise<Worker> {
```
Comment (Arun-KumarH) [inline src/utils.ts:319]: Move this to `scs-jobs` as a seaprate module under RC `libs`.

File: src/scs-jobs/delete_expired_users_job.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import {
```
Comment (Arun-KumarH) [inline src/scs-jobs/delete_expired_users_job.ts:1]: Rename this folder as `jobs`

File: cfg/config.json [hunk 4]
```diff
@@ -380,5 +381,26 @@
         "scope": "email"
       }
     }
+  },
+  "importJobs": {
```
Comment (Arun-KumarH) [inline cfg/config.json:391]: The job is added in the DataImporter - it should be added to this repo - https://github.com/restorecommerce/data/tree/master/demo-shop/data

Comment (vanthome) [reply cfg/config.json:391]: Please rename `unactivatedAccountExpiry` --> `inactivatedAccountExpiry`

File: cfg/config.json [hunk 5]
```diff
@@ -380,5 +381,26 @@
...
+    "jobs": [
+      {
+        "id": "delete-expired-users",
+        "type": "DELETE_EXPIRED_USERS",
```
Comment (vanthome) [inline cfg/config.json:395]: Name `DELETE_USERS_WITH_EXPIRED_ACTIVATION`

**Pull request merged**